dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -244,7 +233,7 @@ export default defineComponent({
           ),
           percentageOfTotalProcurement:
             lksgProcurementCategory.percentageOfTotalProcurement != null
-              ? String(lksgProcurementCategory.percentageOfTotalProcurement)
+              ? String(lksgProcurementCategory.percentageOfTotalProcurement).concat(" %")
```

Comment:
Reminder to self to check out why this exact thing was done when deploying the ticket.